aiven/client/cliarg.py

```diff
@@ -92,6 +92,30 @@ def wrapped(self):
 arg.replication = arg("--replication", type=int, required=True, help="Replication factor")
 arg.retention = arg("--retention", type=int, help="Retention period in hours (default: unlimited)")
 arg.retention_bytes = arg("--retention-bytes", type=int, help="Retention limit in bytes (default: unlimited)")
+arg.tag = arg(
+    "--tag",
+    dest="topic_option_tag",
+    metavar="KEY[=VALUE]",
+    action="append",
+    default=[],
```

Comment:
Having mutable objects in globals may cause weird behavior. I think it is better to leave this with `None` as a default and handle that after parsing the CLI arguments.